src/stream_base-inl.h

```diff
@@ -72,6 +72,10 @@ void StreamBase::GetFD(Local<String> key,
                        const PropertyCallbackInfo<Value>& args) {
   StreamBase* wrap = Unwrap<Base>(args.Holder());
 
+  // Mimic implementation of StreamBase::GetFD() and UDPWrap::GetFD().
+  if (wrap == nullptr)
+    return args.GetReturnValue().Set(-1);
```

Comment:
Sorry. Didn't see what you were saying. Yes, we should cover that as well. Thanks for pointing it out.